### ISSUE TITLE
array api support

### ISSUE THREAD

**Issue by icfaust** [IntelPython/dpnp#1755]
Hello,

Array API support (which is in the docs external links), is currently missing (missing ```__array_namespace__``` in dpnp.ndarray, incorrect signatures in methods (```to_device``` for example).  Will the array_api standard be implemented in dpnp?

**Comment by samir-nasibli** [comment]
@oleksandr-pavlyk @ndgrigorian @antonwolfy is there any updates?

**Comment by ndgrigorian** [comment]
> @oleksandr-pavlyk @ndgrigorian @antonwolfy is there any updates?

Yes, it won't be in this coming release, but dpnp is targeting array API support for the `dpnp` namespace (like NumPy 2.0 and greater, no array API sub-module).

**Comment by icfaust** [comment]
is there a targeted release for array api support?

**Comment by antonwolfy** [comment]
The `0.17.0` release achieves 100% compliance with Python Array API specification (revision [2023.12](https://data-apis.org/array-api/2023.12/)).
Closing the case since the release has been published and available now.
